Modules/@babylonjs/react-native/shared/BabylonNative.cpp

```diff
@@ -33,13 +34,13 @@ namespace Babylon
             (
                 jsiRuntime,
                 jsi::Function::createFromHostFunction(jsiRuntime, jsi::PropNameID::forAscii(jsiRuntime, "executor"), 0, [this](jsi::Runtime& rt, const jsi::Value&, const jsi::Value* args, size_t) -> jsi::Value
+            {
+                m_resolveInitPromise = [&rt, resolve{ std::make_shared<jsi::Value>(rt, args[0]) }]()
                 {
-                    m_resolveInitPromise = [&rt, resolve{ std::make_shared<jsi::Value>(rt, args[0]) }]()
-                    {
-                        resolve->asObject(rt).asFunction(rt).call(rt);
-                    };
-                    return {};
-                })
+                    resolve->asObject(rt).asFunction(rt).call(rt);
+                };
+                return {};
+            })
```

Comment:
Revert? Seems like this got un-indented or something?

Reply:
this should now be fixed